ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/config/WorkflowConfig.scala

```diff
@@ -53,6 +54,7 @@ object WorkflowConfig {
       applicationIds: List[FooSubmissionConfig.ApplicationId] = List.empty,
       maybeWaitForSubmission: Option[Boolean] = None,
       observerPartySets: List[FooSubmissionConfig.PartySet] = List.empty,
+      allowNonTransientContracts: Boolean = false,
```

Comment:
If `allowNonTransientContracts` we still might get transient contracts but will usually get mostly non-transient ones. When it's not set then all contracts that are consumed are transient.
Whether a contract is consumed or not depends on a separate config value which describes the probability of issuing a consuming exercise.

I like `allowNonTransientContracts` because when it's false by default it preserves the previous semantics.